gatsby/content/blog/2022/12/2022-12-07-synapse-1.73.mdx

```diff
@@ -0,0 +1,57 @@
+---
+date: '2022-12-07'
+title: Synapse 1.73 released
+categories:
+- Releases
+author: Mathieu Velten
+---
+
+And here is another update to your beloved Matrix server implementation, [Synapse
+1.73](https://github.com/matrix-org/synapse/releases/tag/v1.73.0).
+
+## Announcements
+
+### Legacy Prometheus metric names removed
+
+As mentioned in previous blogpost, after disabling comes removing !
+
+`enable_legacy_metrics` has been removed, sorry if it breaks your dashboard but
+the upgrade should not be that difficult, and you got some heads-up.
```

Comment:
This sounds unnecessarily aggressive. Maybe start by giving a bit more context on what's going on (some people might be new to Matrix, and not everyone is following every announcement blog posts) and then explain objectively what's happening now, e.g.

> When releasing [Synapse 1.69](https://matrix.org/blog/2022/10/17/synapse-1-69-released#upcoming-removal-of-deprecated-prometheus-metric-names) a couple of months ago, we also announced the removal of old Prometheus metrics that have been replaced by more aptly named ones.
>
> Synapse 1.73 implements the final phase of this plan and entirely removes support for those metrics. As a result, the `enable_legacy_metrics` configuration option, which was introduced in [Synapse 1.71](https://matrix.org/blog/2022/11/22/synapse-1-72-released#announcements), has also been removed.
>
> Server administrators who are still relying on these legacy metric names are encouraged to update their dashboards at their earliest convenience. The list of renamed metrics can be found [here](https://matrix-org.github.io/synapse/v1.73/metrics-howto.html#renaming-of-metrics--deprecation-of-old-names-in-12), and more information can be found in the [upgrade notes](https://matrix-org.github.io/synapse/v1.73/upgrade#legacy-prometheus-metric-names-have-now-been-removed).